Formula/aws-sdk-cpp.rb

```diff
@@ -27,14 +27,16 @@ class AwsSdkCpp < Formula
 
   def install
     ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath}"
-    mkdir "build" do
-      args = %w[
-        -DENABLE_TESTING=OFF
-      ]
-      system "cmake", "..", *std_cmake_args, *args
-      system "make"
-      system "make", "install"
-    end
+    # Avoid OOM failure on Github runner
+    ENV.deparallelize if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
```

Comment:
Wasn't sure if there was a way to detect self-hosted so that it only deparallelizes on GitHub runner.

Current run seemed to work. Hopefully will be able to keep avoiding self-hosted which can get blocked for days. Previous runs used 2 parallel jobs, so deparallelizing adds about 1 hour overhead.